src/v/cluster/members_backend.cc

```diff
@@ -330,12 +339,18 @@ ss::future<std::error_code> members_backend::reconcile() {
     }
 
     // process one update at a time
+    vassert(!_updates.empty(), "_updates was empty");
```

Comment:
Make this a checked deference, since we do suspend above after checking earlier that it was non empty, so we can't prove this safe with local reasoning.